app/components/ui/LandingPageEligibilityBlock.jsx

```diff
@@ -33,6 +33,8 @@ LandingPageEligibilityBlock.props = {
     name: PropTypes.string,
     id: PropTypes.number,
     feature_rank: PropTypes.number,
+    service_count: PropTypes.number,
+    // TODO(cliff): Deprecate resource_count
     resource_count: PropTypes.number,
```

Comment:
You should be able to just delete this from the PropTypes if it's unused. You only need to declare the fields that are actually being used in the PropTypes, and it won't complain about extra things being passed in.